packages/typespec-azure-resource-manager/lib/foundations/backcompat.tsp

```diff
@@ -0,0 +1,40 @@
+namespace Azure.ResourceManager.Foundations {
+  // extended-location
+  alias ExtendedLocation = CommonTypes.ExtendedLocation;
+  alias ExtendedLocationType = CommonTypes.ExtendedLocationType;
+
+  // managed-identities
+  alias ManagedServiceIdentity = CommonTypes.ManagedServiceIdentity;
+  #suppress "deprecated" "Need for back compatibility in common-types"
```

Comment:
Suppression should have the user action, like use the ManagedServiceIdentityProperty mix-in instea

Reply:
Removed. The CommonType.UserAssignedIdentities has #deprecate which will surface up on the alias.